tests/Fixer/Whitespace/MethodChainingIndentationFixerTest.php

```diff
@@ -281,6 +315,121 @@ public function testFix73(): void
         );
     }
 
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideFix74Cases
+     * @requires PHP 7.4
```

Comment:
PHP 7.4 is now the minimum required version of PHP, this requirement is superfluous.

Reply:
Tests moved to `testFix` which does not have the requirement